x-pack/plugin/identity-provider/docs/en/rest-api/idp-saml-init.asciidoc

```diff
@@ -0,0 +1,111 @@
+[role="xpack"]
+[[idp-saml-init]]
+=== Generate a SAML authentication response message
+++++
+<titleabbrev>Generate SAML response</titleabbrev>
+++++
+Generates a SAML Response message to be sent to a Service Provider
+
+[[idp-saml-init-request]]
+==== {api-request-title}
+
+`POST /_idp/saml/init`
+
+[[idp-saml-init-prereqs]]
+==== {api-prereq-title}
+
+* To use this API, you must have a role that grants the `cluster:admin/idp/saml/init` privilege.
+
+[[idp-saml-init-desc]]
+==== {api-description-title}
+
+This API generates a SAML Response message that should be sent to a Service Provider as part of an
+IDP initiated or SP initiated SAML Single Sign On. This API expects the caller to present
+credentials for the user that the SAML Response will be created for as "Secondary Authentication"
+using the `es-secondary-authorization` HTTP Request header.
+
+The SAML response is returned as an XML String and the caller of the API is responsible to instruct
+the end user's browser to make an HTTP Post request to the Service Provider with the SAML response
+Base64 and URL encoded.
+
+[[idp-saml-init-body]]
+==== {api-request-body-title}
```

Comment:
Is there a `acs` body param?

https://github.com/elastic/elasticsearch/pull/54215/files#diff-1cd906e941a844a669917a5383d63b97R127

Reply:
Hmm, in `7.7-SNAPSHOT` if I pass the `acs` body param I get

```json
{
	"error": {
		"root_cause": [{
			"type": "x_content_parse_exception",
			"reason": "[1:34] [idp_init_sso] unknown field [acs]"
		}],
		"type": "x_content_parse_exception",
		"reason": "[1:34] [idp_init_sso] unknown field [acs]"
	},
	"status": 400
}
```

Reply:
The snapshot you are using does not contain https://github.com/elastic/elasticsearch/commit/5434a3c6714596b3947de3605d93285df5507ad6 ( was merged 6 hours ago) in which this change is added in 7.7 
These docs need updating after recent code changes Lloyd, I'll update the PR today and ping you when it's ready

Reply:
Ah, thanks, just got a newer snapshot and it works now with the `acs` param.